src/subscription/subscribe.js

```diff
@@ -170,21 +193,22 @@ export function createSourceEventStream(
   const fieldNodes = fields[responseName];
   const fieldNode = fieldNodes[0];
   const fieldDef = getFieldDef(schema, type, fieldNode.name.value);
-  invariant(
-    fieldDef,
-    'This subscription is not defined by the schema.'
-  );
+  if (!fieldDef) {
+    return emptyAsyncIterator();
```

Comment:
In the doc block above:

> If an error is encountered during creation, an Error is thrown.

Why shouldn't we treat a missing fieldDef as an error?